app/local_setup.sh

```diff
@@ -5,4 +5,5 @@ python manage.py migrate && \
 python manage.py update_assettype && \
 python manage.py loaddata 'fixtures/multivector_fixture.json' && \
 echo yes | python manage.py collectstatic && \
+pre-commit install && \
```

Comment:
Is it added to the dependencies the developpers need to install in the README steps? If not can you add a it there?

Reply:
Good catch, I did forget to update the readme.